.github/workflows/publish.yml

```diff
@@ -95,13 +95,34 @@ jobs:
       - name: Verify demo browser npm version and run deployment script
         run: |
           demo_current_version=$(.github/script/get-demo-browser-current-version)
+          demo_current_version=$(echo $demo_current_version | tr -d '^~*')
           echo "Current demo version:" $demo_current_version
+
+          echo "Current released npm version:" ${{ steps.npm_version.outputs.npm_version }}
           if [[ *$demo_current_version* != "^#${{ steps.npm_version.outputs.npm_version }}" ]];
           then
             echo "Demo npm version is not the same as the current released version"
             exit 1
           else
             npm run build --app=meetingReadinessChecker
             script/github-action-awscli-installation
-            script/deploy-canary-demo
+            
+            npm_latest_version=$(npm view amazon-chime-sdk-js@latest version)
```

Comment:
Can we do this in node script? Who want to read bash script :D

Reply:
Sure